test/http-client-tests/src/main/java/software/amazon/awssdk/http/SdkHttpClientTestSuite.java

```diff
@@ -218,6 +221,48 @@ public void testCustomTlsTrustManagerAndTrustAllFails() throws Exception {
         assertThatThrownBy(() -> createSdkHttpClient(httpClientOptions)).isInstanceOf(IllegalArgumentException.class);
     }
 
+    @Test
```

Comment:
Adding test cases in SdkHttpClientTestSuite makes sure that all the other SDK clients gets testes since this class is extended by other HTTP clients too.

Reply:
done